features/support/env.rb

```diff
@@ -94,5 +95,11 @@
   create(:corporate_partner)
 end
 
+Before('@auth-required') do
+  allow(Authenticable).to receive(:username)  { ENV['AUTH_USERNAME'] }
+  allow(Authenticable).to receive(:password)  { ENV['AUTH_PASSWORD'] }
```

Comment:
Argh yeah I forgot to replace those with some explicit fake data I was going to pass in. Thanks for spotting this.